src/Umbraco.Web/PublishedCache/NuCache/DataSource/LazyCompressedString.cs

```diff
@@ -10,11 +10,12 @@ namespace Umbraco.Web.PublishedCache.NuCache.DataSource
     /// Lazily decompresses a LZ4 Pickler compressed UTF8 string
     /// </summary>
     [DebuggerDisplay("{Display}")]
-    internal struct LazyCompressedString
+    public struct LazyCompressedString
```

Comment:
need to be public?

Reply:
Yes in order to allow changing the serializer